climada/hazard/tc_clim_change.py

```diff
@@ -119,17 +155,23 @@ def calc_scale_knutson(ref_year=2050, rcp_scenario=45):
     """Comparison 2081-2100 (i.e., late twenty-first century) and 2001-20
     (i.e., present day). Late twenty-first century effects on intensity and
     frequency per Saffir-Simpson-category and ocean basin is scaled to target
-    year and target RCP proportional to total radiative forcing of the respective
-    RCP and year.
-
-    Parameters:
-        ref_year (int): year between 2000 ad 2100. Default: 2050
-        rcp_scenario (int):  26 for RCP 2.6, 45 for RCP 4.5 (default),
-            60 for RCP 6.0 and 85 for RCP 8.5.
-
-    Returns:
-        float
+    year and target RCP proportional to total radiative forcing of the
+    respective RCP and year.
+
+    Parameters
+    ----------
+    ref_year : int, optional
+        year between 2000 ad 2100. Default: 2050
+    rcp_scenario: int, optional
+        26 for RCP 2.6, 45 for RCP 4.5. The default is 45
+        60 for RCP 6.0 and 85 for RCP 8.5.
+
+    Returns
+    -------
+        : float
+        factor to scale Knuston parameters to the give RCP and year
     """
+
```

Comment:
There is a convention not to put any blank lines before or after function docstrings:
http://www.pydocstyle.org/en/stable/error_codes.html (D202)
https://stackoverflow.com/questions/18914108/python-pep-blank-line-after-function-definition
https://pandas.pydata.org/docs/development/contributing_docstring.html#general-rules
https://www.python.org/dev/peps/pep-0257/
```suggestion
```

Reply:
Yeahhh I know, but it is so ugly ... but well, it is convention.